packages/salesforcedx-sobjects-faux-generator/src/describe/sObjectDescribe.ts

```diff
@@ -14,41 +14,41 @@ import { xhr, XHROptions, XHRResponse } from 'request-light';
 import { CLIENT_ID } from '../constants';
 
 export interface SObject {
-  actionOverrides: any[];
-  activateable: boolean;
-  childRelationships: ChildRelationship[];
-  compactLayoutable: boolean;
-  createable: boolean;
-  custom: boolean;
-  customSetting: boolean;
-  deletable: boolean;
-  deprecatedAndHidden: boolean;
-  feedEnabled: boolean;
+  actionOverrides?: any[];
```

Comment:
Could you elaborate on the reasoning for making these optional?

Reply:
I wanted to take advantage of a lot of the existing generator code that was built around the Field interface. When creating Field objects during local generation, we don't have all of the extra information that comes from a describe call. I don't see the purpose of having a lot of these fields as required if so many of them aren't actually being used by the generation process.